packages/flutter/lib/src/gestures/converter.dart

```diff
@@ -44,7 +46,16 @@ class _PointerState {
 class PointerEventConverter {
   PointerEventConverter._();
 
+  /// Clears internal state mapping platform pointer identifiers to
+  /// [PointerEvent] pointer identifiers.
+  ///
+  /// Visible only so that tests can reset the global state contained in
+  /// PointerEventConverter.
```

Comment:
if you want that second paragraph to be in the documentation, it should be [PointerEventConverter].